### ISSUE TITLE
Remove deprecated twistd mail --passwordfile option

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5541]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)| @cyli reported|
|-|-|
|Trac ID|trac#5541|
|Type|enhancement|
|Created|2012-03-13 10:14:55Z|
|Branch|https://github.com/twisted/twisted/tree/mail-passwordfile-5541|

It has been deprecated since 11.0 by [#4740](https://github.com/twisted/twisted/issues/4740), and has been broken for the last year anyway (see [#5219](https://github.com/twisted/twisted/issues/5219)) - perhaps it should be removed.

If this is incorrect, perhaps it should be re-deprecated and tests for backwards-compatibility should be added.

Attachments:

* [5541.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d13/d13b34684c44f4565d668ade3d37e98a2c5e4116/a59e48f44367bf21a28461aaa4d3c24aa146792d.patch) (2213 bytes) - added by moijes12 on 2012-06-13 07:23:43Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5541 5541
type__enhancement enhancement
reporter__cyli cyli
priority__normal normal
milestone__ 
branch__branches_mail_passwordfile_5541 branches/mail-passwordfile-5541
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1331633695000000 1331633695000000
changetime__1339947053000000 1339947053000000
version__None None
owner__therve therve
cc__moijes12
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)<a name="note_2"></a>|@moijes12 commented|
|-|-|

I tried this using bazaar. I hope its ok.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve commented|
|-|-|

(In [34572]) Branching to 'mail-passwordfile-5541'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @therve|
|-|-|

Looking good!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [34574]) Merge mail-passwordfile-5541

Author: moijes12
Reviewer: therve
Fixes: [#5541](https://github.com/twisted/twisted/issues/5541)

Remove the --passwordfile option of twistd mail.
